release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -993,21 +1092,190 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
+
+* Previously the pruner was trying to detect the registry name using image streams, but when there were no image streams the pruner failed to detect the registry name. With this fix, the Image Registry Operator provides the pruner with the registry name if the registry is configured or disables registry pruning if the registry is not installed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888494[*BZ#1888494*])
+
+* Previously, there was a lack of analysis on Operand deployment status when defining the Operator status. This meant that in some scenarios the Image Registry Operator was presenting itself with two contradicting pieces of information. It was informing the user that it was not Available and at the same time not Degraded. These two conditions were still being presented even after the deployment stopped trying to get the image registry up and running. In this scenario the Degraded flag should be set by the Operator. By taking image registry deployment into account, the Operator now sets itself to Degraded if the Operand deployment reaches its progress deadline when trying to get the application running. Now when the Deployment fails, after the progress deadline has been reached, the Operator sets itself to Degraded. The Operator still reports itself as Progressing while the Operator deployment is progressing. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1889921[*BZ#1889921*])
+
+* Previously the Image Registry Operator did not use its entrypoint because an explicit command was provided. So a cluster-wide `trusted-ca` was not used by the Operator and the Operator could not connect to storage providers that do not work without custom `trusted-ca`. This fix removed the explicit command from the pod spec. Now the image entrypoint is used by the container that applies `trusted-ca`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892799[*BZ#1892799*])
+
+* Previously the default log level for the pruner was `2`. So when an error happened, the pruner was dumping stack trace. This fix changed the default log level to `1`. Now only the error message is printed without stack traces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894677[*BZ#1894677*])
+
+* Previously the `configs.imageregistry.operator.openshift.io` status field did not update during the Operator sync, which meant the status field was not presenting the most up to date applied swift configuration. With this fix the sync process updates the `configs.imageregistry.operator.openshift.io` status to the spec values. The spec and status fields are in sync with the status field, presenting the applied configuration. link:https://bugzilla.redhat.com/show_bug.cgi?id=1907202[*BZ#1907202*])
```

Comment:
Just a suggestion that it might be worth and search a replace of replacing:
`With this fix `
with:
`With this fix, '

Reply:
And similarly, 
Previously <- space, no comma
with
Previously, <- comma and then space